pkgs/ok_http/lib/src/ok_http_client.dart

```diff
@@ -21,6 +21,49 @@ import 'package:jni/jni.dart';
 
 import 'jni/bindings.dart' as bindings;
 
+/// Configurations for the [OkHttpClient].
+class OkHttpClientConfiguration {
+  /// The maximum duration (in milliseconds) to wait for a call to complete.
+  ///
+  /// If a call does not finish within the specified time, it will throw a
+  /// `ClientException` with the message "java.io.InterruptedIOException...".
+  /// Default is set to `0`, which indicates no timeout.
+  ///
+  /// See [OkHttpClient.Builder.callTimeout](https://square.github.io/okhttp/5.x/okhttp/okhttp3/-ok-http-client/-builder/call-timeout.html).
+  final int callTimeout;
+
+  /// The maximum duration (in milliseconds) to wait while connecting a TCP
+  /// Socket to the target host.
+  ///
+  /// Default is set to `10000` milliseconds.
+  ///
+  /// See [OkHttpClient.Builder.connectTimeout](https://square.github.io/okhttp/5.x/okhttp/okhttp3/-ok-http-client/-builder/connect-timeout.html).
+  final int connectTimeout;
+
+  /// The maximum duration (in milliseconds) to wait for a TCP Socket and for
+  /// individual read IO operations.
+  ///
+  /// Default is set to `10000` milliseconds.
+  ///
+  /// See [OkHttpClient.Builder.readTimeout](https://square.github.io/okhttp/5.x/okhttp/okhttp3/-ok-http-client/-builder/read-timeout.html).
+  final int readTimeout;
+
+  /// The maximum duration (in milliseconds) to wait for individual write IO
+  /// operations.
+  ///
+  /// Default is set to `10000` milliseconds.
+  ///
+  /// See [OkHttpClient.Builder.writeTimeout](https://square.github.io/okhttp/5.x/okhttp/okhttp3/-ok-http-client/-builder/write-timeout.html).
+  final int writeTimeout;
+
+  const OkHttpClientConfiguration({
```

Comment:
Is there a way to test this? Maybe a pattern like this would work:
https://github.com/dart-lang/http/blob/master/pkgs/cronet_http/example/integration_test/cronet_configuration_test.dart

Reply:
I was able to add a few useful cases for `callTimeout`. Unsure of a good way to simulate read, write and connect timeouts however.